pkg/inventory/inventory.go

```diff
@@ -9,7 +9,7 @@
 // objects, we can correctly prune and teardown sets
 // of resources.
 
-package prune
+package inventory
```

Comment:
Shouldn't we also move `GetPreviousInventoryObjects` into the inventory package? It doesn't seem like it belongs with prune.

Reply:
Exactly. In future PR, there will be `InventoryClient` interface, which will implement this (unit tests are easier too).